openedx/core/djangoapps/credit/api.py

```diff
@@ -322,6 +322,93 @@ def get_credit_requests_for_user(username):
     return CreditRequest.credit_requests_for_user(username)
```

Comment:
@wedaly OK. I should be at my desk most of the afternoon - feel free to stop by whenever.